contracts/src/TeleporterTokenSource.sol

```diff
@@ -417,12 +436,25 @@ abstract contract TeleporterTokenSource is
             uint256 sourceAmount =
                 _processReceivedTransfer(sourceBlockchainID, originSenderAddress, payload.amount);
 
+            DestinationBridgeSettings memory originSettings =
```

Comment:
Can we rename `originSettings` to `destinationSettings` to remove the directionality from the name?

Reply:
Makes sense, thanks for clarifying